packages/plugin-core/src/services/TextDocumentService.ts

```diff
@@ -279,6 +283,16 @@ export class TextDocumentService implements ITextDocumentService {
     });
   }
 
+  public containsFrontmatter(textDocument: TextDocument) {
```

Comment:
This function seems stateless?  How about making it static and removing it from the `ITextDocumentService` interface?

Because this is stateless fn, I think it makes more sense to be called statically rather than injecting an instance into `WorkspaceWatcher`

Reply:
Sounds good to me